.rspec

```diff
@@ -0,0 +1,2 @@
+--color
```

Comment:
Have we been checking the `.rspec` file into the other extracted modules? Seems like it should be part of the `.gitignore` so people working on the module can have whatever they prefer in there.

Reply:
This file is added by PDK, so yes, it's in all the other modules